Sources/MongoSwift/BSON/Document.swift

```diff
@@ -540,3 +540,10 @@ extension Document: ExpressibleByArrayLiteral {
         self.init(elements)
     }
 }
+
+// An extension of `Document` to add the capability to be hashed
+extension Document: Hashable {
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(self.rawBSON)
+    }
+}
```

Comment:
[nit] add a newline at the end of the file.

As part of the workflow, you should make sure both that the tests pass locally and that swiftlint didn't detect any violations before opening a PR. It helps reduce the number of round trips in code review and the time it takes code to get merged.